framework/src/main/java/org/checkerframework/common/wholeprograminference/scenelib/AMethodWrapper.java

```diff
@@ -0,0 +1,122 @@
+package org.checkerframework.common.wholeprograminference.scenelib;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.HashMap;
+import java.util.Map;
+import javax.lang.model.element.Name;
+import javax.lang.model.type.TypeMirror;
+import org.checkerframework.checker.signature.qual.DotSeparatedIdentifiers;
+import org.checkerframework.checker.signature.qual.FullyQualifiedName;
+import scenelib.annotations.el.AMethod;
+
+/**
+ * A wrapper for the AMethod class from scenelib. Keeps more information on the return type, and
```

Comment:
I found this a bit confusing.  I would change "on" to "about" to clarify that you are not talking about *where* information is  stored, but *what* information is stored.